advanced/Scripts/database_migration/gravity/2_to_3.sql

```diff
@@ -0,0 +1,61 @@
+.timeout 30000
+
+PRAGMA FOREIGN_KEYS=OFF;
+
+ALTER TABLE regex RENAME TO regex_blacklist;
```

Comment:
I don't think SQLite will automatically wrap these into a transaction, so we should explicitly start a transaction here. This will avoid partially-upgraded databases if an error occurs during the upgrade process.